app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UsersController < ActionController
+  def index; end
+
+  def show; end
+end
```

Comment:
- [ ] Similarly, the `UserController`  should inherits from `ApplicationController`, and our `UserController` class should be similar to:

```ruby
# app/controllers/users_controller.rb
class UsersController < ApplicationController
  
  def index; end

  def show; end

end
```

Reply:
well resolved! ✔️